lib/showoff.rb

```diff
@@ -407,11 +408,8 @@ def process_markdown(name, section, content, opts={:static=>false, :pdf=>false,
 
         # Apply the template to the slide and replace the key to generate the content of the slide
         sl = process_content_for_replacements(template.gsub(/~~~CONTENT~~~/, slide.text))
-        sl = Tilt[:markdown].new(nil, nil, engine_options) { sl }.render
```

Comment:
What do you think of moving it to the next method? It seems like it would make sense there.